recipes/ninja/all/conanfile.py

```diff
@@ -1,6 +1,11 @@
 from conan import ConanFile
 from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
 from conan.tools.files import copy, get
+from conan.tools.scm import Version
+try:
+    from conan import conan_version
+except ImportError
+    from conans import __version__ as conan_version
```

Comment:
https://github.com/conan-io/conan-center-index/issues/12124#issuecomment-1210327220

Reply:
It seems like many of these issues result from trying to run Conan 2.0 recipes through Conan 1.x. Why did CCI not elect to simply create a parallel branch for 2.0 work analogous to how the Conan project divides its efforts.  Adding 1.x workarounds like this significantly detract from the simplicity and elegance of pure 2.0 recipes. There appear to be quite a few issues like this where there is no single solution that works for both 1,x and 2.0.

Reply:
@System-Arch All recipes will be migrated and CCI will start to adopt Conan 2.x more and more. Please, check the roadmap: https://github.com/conan-io/conan-center-index/blob/master/docs/v2_roadmap.md

Reply:
Thanks for the links and opening that issue @SpaceIm. Everyone in agreement on me closing this PR then?

Reply:
@jwillikers yes please.